.github/workflows/upload_binary.yml

```diff
@@ -29,10 +28,10 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Set up Python ${{ matrix.python-version }}
+      - name: Set up latest Python
         uses: actions/setup-python@v2
         with:
-          python-version: ${{ matrix.python-version }}
+          python-version: "*"
```

Comment:
I leave this line out in other CI jobs but I guess this is more explicit wildcard …